test/plugins/scenarios/go-redisv9/config/excepted.yml

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+  - serviceName: go-redisv9
+    segmentSize: ge 1
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: set
+            parentSpanId: 0
+            spanId: 1
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:6379
+            skipAnalysis: false
+            tags:
+              - {key: cache.type, value: redis}
+              - {key: cache.args, value: not null}
+          - operationName: get
+            parentSpanId: 0
+            spanId: 2
+            spanLayer: Cache
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5014
+            isError: false
+            spanType: Exit
+            peer: redis-server:6379
+            skipAnalysis: false
+            tags:
+              - {key: cache.type, value: redis}
+              - { key: cache.args, value: not null}
+          - operationName: redis.pipeline set set
```

Comment:
don't add a blank in an operation name, split by "/" is a good choice.

Reply:
So the pipeline operation will shows up like `redis/pipeline/set/set/get/hmset/...`. Wondering if we could have a better format for op name with multiple commands.

I'm changing to this format for now. But feel free to comment with other idea before we merge this into master.

Reply:
This would be a good refer, https://github.com/apache/skywalking-java/blob/main/test/plugin/scenarios/jedis-4.x-scenario/config/expectedData.yaml

After all, Java plugins are existing for a long time.

Reply:
Batch is tricky actually, and easy to cause overload.

If I read the codes correctly, https://github.com/apache/skywalking-java/blob/6621f82739340bbaaac2338cb06ccbe3d38513a5/apm-sniffer/apm-sdk-plugin/jedis-plugins/jedis-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jedis/v4/JedisMethodInterceptor.java#L31

Reply:
well, based on discussion above I am trying to proposal span like this:

```
  - operationName: redis/pipeline  # could we just use pipeline directly, it's clear in "redis context"
    parentSpanId: 0
    spanId: 4
    spanLayer: Cache
    startTime: nq 0
    endTime: nq 0
    componentId: 5014
    isError: false
    spanType: Exit
    peer: redis-server:6379
    skipAnalysis: false
    tags:
      - {key: cache.type, value: redis}
      - {key: cache.cmd, value: 'pipeline:get/get'}  # retain necessary information for debugging
```

Reply:
`pipeline:get/get` Why `get/get`?

Reply:
> `pipeline:get/get` Why `get/get`?

because two different command are executed with `pipeline`. In this test case I execute the following codes:
```
# start pipeline
GET xxxx
GET yyyy
# submit my pipline for execution
```

Other example of the tag value:
```
pipeline:set/get/get/get/set

# when len(op) > 10
pipeline:hmset/get/set/get/get/get/get/get/get/get/...
```